chain-db-watcher/src/generated/discussion-db-graphql.ts

```diff
@@ -1136,52 +759,17 @@ export type CouncilMemberCouncilsArgs = {
   where?: Maybe<CouncilWhereInput>;
 };
 
-export type CouncilMemberConnection = {
```

Comment:
All these removal are linked to the fact that the chain-db watcher was connected to the server (port 4000) and it does not have access to all the creation mutation. This is correct. Those generated files must have been created pointing the node-watcher deployment itself.